pkg/skaffold/runner/runner_test.go

```diff
@@ -172,17 +172,28 @@ func (t *TestBench) Actions() []Actions {
 	return append(t.actions, t.currentActions)
 }
 
-func (t *TestBench) WatchForChanges(context.Context, io.Writer, func(context.Context, io.Writer) error) error {
+func (t *TestBench) WatchForChanges(ctx context.Context, out io.Writer, checker func() needs, devLoop func(context.Context, io.Writer, needs) error) error {
```

Comment:
I really don't like this method here - it is super fragile, we have to reimplement it any time the Monitor logic is changing :(